doc/releases/changelog-0.42.0.md

```diff
@@ -4,98 +4,117 @@
 
 <h3>New features since last release</h3>
 
-* Leveraging quantum just-in-time compilation to optimize parameterized hybrid workflows with the quantum 
-  natural gradient optimizer is now possible with the new :class:`~.QNGOptimizerQJIT` optimizer. 
-  [(#7452)](https://github.com/PennyLaneAI/pennylane/pull/7452)
+<h4>State-of-the-art templates and decompositions 🐝</h4>
+
+* A new decomposition using [unary iteration](https://arxiv.org/pdf/1805.03662) has been added to :class:`~.Select`.
+  This state-of-the-art decomposition reduces the :class:`~.T`-count significantly, and uses :math:`c-1` auxiliary wires,
+  where :math:`c` is the number of control wires. 
+  [(#7623)](https://github.com/PennyLaneAI/pennylane/pull/7623)
+  [(#7744)](https://github.com/PennyLaneAI/pennylane/pull/7744)
+  [(#7842)](https://github.com/PennyLaneAI/pennylane/pull/7842)
+
+  Unary iteration leverages auxiliary wires to store intermediate values for reuse among the 
+  different multi-controlled operators, avoiding unnecessary recomputation and leading to more 
+  efficient decompositions to elementary gates. 
   
-  The :class:`~.QNGOptimizerQJIT` optimizer offers a `jax.jit`- and `qml.qjit`-compatible analogue to the existing 
-  :class:`~.QNGOptimizer` with an Optax-like interface:
+  This state-of-the-art decomposition rule for :class:`~.Select` is available with the graph-based
+  decomposition system enabled via :func:`~.decomposition.enable_graph`:
 
   ```python
   import pennylane as qml
```

Comment:
What's our policy on "import pennylane" in the changelog? We're currently importing it in several code blocks. Not a problem and makes it easier to copy-paste code blocks. But we don't import it in the docs :thinking:

Reply:
Usually I just have it in the very first code block in the chnagelog and that's it 🤔

Reply:
Will make this update then :+1: